src/components/SearchResults/SearchResults.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+import { users as fakeUsers} from '../../assets/fake-users'
+import { UserResults } from '../../assets/definitions'
+import ApplicantPreview from '../ApplicantPreview/ApplicantPreview'
+import './SearchResults.scss'
 
-const SearchResults: React.FC = () => {
-return <div>Hello World</div>
+interface Props extends RouteComponentProps {}
+
+const SearchResults: React.FC<Props> = (props) => {
+  const [users, setUsers] = useState<Array<UserResults>>([])
+  useEffect(() => { 
+    fakeUserFetch()
+    .then(data => setUsers(data))
+  }, [])
+
+  const fakeUserFetch = async () => {
+    return await fakeUsers
+  }
+  
+  let userData = users.map((user) => {
+    return ( 
+      <ApplicantPreview 
+        id={user.id}
+        username={user.username}
+        bio={user.bio}
+        updated_at={user.updated_at}
```

Comment:
There was a comment from BE that the updated_at was giving them some trouble. We may need to table this and come back to it later.